README.md

```
@@ -1,126 +1,221 @@
-        - Main
-        - Career
-    - B/E 신규개발 진행
-        - Spring-Boot 프로젝트 생성
-        - Spring Security 및 JWT 등 보안 설정
-        - JPA 설정 및 Entity 생성
-        - 컨트롤러 및 서비스 파일 구성
-    - AWS 인프라 EC2 셋팅
-        - F/E 배포
+# 🚀 basilry.kim | 프론트엔드
+
+<p align="center">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)